### DISCUSSION TITLE
User Aliases

### DISCUSSION THREAD

**Discussion by ghost** [langfuse/langfuse#3618] [Ideas]
### Describe the feature or potential improvement

Hi! Love the userId feature. 
An improvement that could be made to this is to allow storing aliases/labels for the users, such that they can be identified more easily when looking in the UI.

**Comment by marcklingen** [comment]
thanks for sharing, can you add your upvote/comments to the main discussion on this here? https://github.com/orgs/langfuse/discussions/1154

**Reply by ghost** [reply]
Thanks Marc!  
Didn't see the existing thread before, I'll upvote and comment there.

**Reply by marcklingen** [reply]
thanks, no worries! happy to help merge these
